### PR TITLE
add: livck-installation tutorial

### PR DESCRIPTION
**Pull request by Rene-Roscher**

### CODE REVIEW

File: tutorials/install-livck/01.en.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+---
+SPDX-License-Identifier: MIT
+path: "/tutorials/install-livck-on-ubuntu-20-04"
+slug: "how-to-install-livck-on-hetzner-cloud"
```
Comment (rook59) [inline tutorials/install-livck/01.en.md:4]: slug has to be the path without /tutorials/

File: tutorials/install-livck/01.en.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
+---
+SPDX-License-Identifier: MIT
+path: "install-livck-on-ubuntu-20-04"
+slug: "how-to-install-livck-on-hetzner-cloud"
```
Comment (rook59) [inline tutorials/install-livck/01.en.md:4]:
```suggestion
path: "/tutorials/install-livck-on-ubuntu-20-04"
slug: "install-livck-on-ubuntu-20-04"
```

File: tutorials/install-livck/01.en.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+DB_USERNAME=root
+DB_PASSWORD="password"
+```
+
+### Step 4 - `Install LIVCK`
+
+Now we install the required packages for the software.
+
+```shell
+composer install --no-dev --optimize-autoloader --ignore-platform-reqs && chmod 777 * -R
```
Comment (rook59) [inline tutorials/install-livck/01.en.md:125]: > chmod 777 * -R

This does not look good to me but seems but be part of the official installation guide https://help.livck.com/#_2-download-files

**Review by rook59 — COMMENTED** [review]
@leojakeson FYI I can't actually test the livck software because it's paid software but the tutorial looks good compared to the installation guide at https://help.livck.com/

File: tutorials/install-livck/01.en.md (hunk 3 above)
Comment (Rene-Roscher) [reply tutorials/install-livck/01.en.md:125]: Hey, if you need to test the entire installation, I can give you a license too. For this you should only have an account and give me your email

Comment (rook59) [reply tutorials/install-livck/01.en.md:125]: Thanks for the offer. I saw you are the owner of this software, so I guess you know how to install it ;) Overall, the tutorial looks good to me. We can leave the chmod as it is, but I am quite sure that such high permissions are not necessary and were only set out of laziness(? no hard feelings).

Comment (Rene-Roscher) [reply tutorials/install-livck/01.en.md]: Laziness has been corrected 😅

**Comment by Rene-Roscher** [comment]
When will the guide be published?

File: tutorials/install-livck/01.en.md (hunk 1 above)
Comment (rook59) [reply tutorials/install-livck/01.en.md]: This is still wrong. Correct path and slug would be for example:
```
path: "/tutorials/install-livck-on-ubuntu-20-04"
slug: "install-livck-on-ubuntu-20-04"
```

Comment (Rene-Roscher) [reply tutorials/install-livck/01.en.md]: Is corrected 🤝

**Review by rook59 — APPROVED** [review]
LGTM

File: tutorials/install-livck/01.en.md (hunk 3 above)
Comment (rook59) [reply tutorials/install-livck/01.en.md]: You might want to update your official docs too ;)

Comment (Rene-Roscher) [reply tutorials/install-livck/01.en.md]: Hey, there is already a completely new page in the starting blocks, which we still have to expand.

**Pull request merged**